table/iterator.go

```diff
@@ -204,10 +205,25 @@ func (itr *Iterator) seekHelper(blockIdx int, key []byte) {
 		return
 	}
 	itr.bi.setBlock(block)
-	itr.bi.seek(key)
+	itr.bi.seek(key, 0)
 	itr.err = itr.bi.Error()
 }
 
+func (itr *Iterator) seekFromOffset(blockIdx int, offset int, key []byte) {
+	itr.bpos = blockIdx
+	block, err := itr.t.block(blockIdx)
+	if err != nil {
+		itr.err = err
+		return
+	}
+	itr.bi.setBlock(block)
+	itr.bi.setIdx(offset)
+	if y.CompareKeys(itr.bi.key, key) >= 0 {
+		return
+	}
```

Comment:
We can return a bool to indicate fallback, let `PointGet` return fallback, then we don't need to change `blockIterator.seek`.

Reply:
Or just ignore the `offset`, just use the old `seek`.